packages/web/src/electron.js

```diff
@@ -431,34 +436,25 @@ ipcMain.on('update', async (event, arg) => {
 })
 
 ipcMain.on('web-update', async (event, arg) => {
-  if (
-    fs.existsSync(path.resolve(app.getAppPath(), `web-update/${buildName}`))
-  ) {
+  const buildPath = appDataPath(`web-update/packages/web/${buildName}`)
```

Comment:
Not sure why `packages/web` wasn't needed before?

Reply:
🤯